SpatialGDK/Documentation/index.md

```diff
@@ -1,11 +1,18 @@
-# The SpatialOS Unreal GDK (pre-alpha)
+# Welcome to the SpatialOS GDK for Unreal 
+
+<img src="{{assetRoot}}assets/unrealgdk-headline-image.png" style=" margin: 0 auto; display: block; width: 60%" />
+
+<br/>
 
 <%(TOC)%>
 
-Welcome to the SpatialOS and Unreal GDK community.
+The SpatialOS Game Development Kit (GDK) for Unreal is a <a href="" data-track-link="Plugin Link Clicked|product=Docs|platform=Win" target="_blank">plugin</a> (Unreal documentation) which allows you to host your game and combine multiple dedicated server instances across one seamless game world, whilst using the Unreal Engine networking API. 
```

Comment:
Can we have "(Unreal documentation)" in the link - this is now our standard until we can get something groovier.

Reply:
This sentence has been bugging me every time I read it. It's got too many elements to it for my tiny brain to take in in one go.
Suggest split into two:
The SpatialOS Game Development Kit (GDK) for Unreal is a <a href="" data-track-link="Plugin Link Clicked|product=Docs|platform=Win" target="_blank">plugin (Unreal documentation)</a>. It allows you to host your game and combine multiple dedicated server instances across one seamless game world whilst using the Unreal Engine networking API.